flight/util/json_parsing.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+
+def waypoint_parsing(filename: str):
+    f = open(filename, )
+    data_set = json.load(f)
+    # print(data_set)
+    f.close()
+
+    waypoint_Locs = []
+
+    for i in range(0, len(data_set["waypoints"])):
+        waypoint_Locs.append(data_set["waypoints"][i])
+
+    return waypoint_Locs
+
+
+def stationary_obstacle_parsing(filename: str):
```

Comment:
please annotate your return type and initialized variables :)
also you're missing a docstring here too